libr/core/cmd_eval.c

```diff
@@ -301,11 +309,11 @@ static int cmd_eval(void *data, const char *input) {
 				nextpal (core, 'j');
 			} else if (input[2] == ' ') {
 				bool failed = false;
-				char *home, path[512];
-				snprintf (path, sizeof (path), ".config/radare2/cons/%s", input + 3);
-				home = r_str_home (path);
-				snprintf (path, sizeof (path), R2_DATDIR"/radare2/"
-					R2_VERSION"/cons/%s", input + 3);
+				char *home, *path, tmp[512];
+				snprintf (tmp, sizeof (tmp), ".config/radare2/cons/%s", input + 3);
```

Comment:
use r_str_newf. keep strings processing simple, unless it really matters for performance